metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -149,10 +152,9 @@ CompletableFuture<MigrationDriverState> migrationState() {
     }
 
     private void recoverMigrationStateFromZK() {
-        log.info("Recovering migration state from ZK");
-        applyMigrationOperation("Recovery", zkMigrationClient::getOrCreateMigrationRecoveryState);
+        applyMigrationOperation("Recovering migration state from ZK", zkMigrationClient::getOrCreateMigrationRecoveryState);
         String maybeDone = migrationLeadershipState.zkMigrationComplete() ? "done" : "not done";
-        log.info("Recovered migration state {}. ZK migration is {}.", migrationLeadershipState, maybeDone);
+        log.info("ZK migration is {}.", maybeDone);
```

Comment:
I'd really prefer to say something like "Initial ZK load is done" / "Initial ZK load is not done"

We should also change `ZkMigrationLeadershipState.zkMigrationComplete` -> `ZkMigrationLeadershipState.initialZkLoadComplete`

After all, the whole process here is technically "ZK migration" not just the initial load